src/SonarQube.Client/Api/Requests/V7_20/GetIssuesRequest.cs

```diff
@@ -61,7 +61,10 @@ private static SonarQubeIssueResolutionState ParseResolutionState(string resolut
         {
             switch (resolution)
             {
-                case "":
+                // Issues with Status=OPEN will not have a resolution so we're
+                // expecting null, but we'll be defensive and handle empty too
+                case null:
```

Comment:
This was the bug - the JSON deserializer used with the non-deprecated API handles missing values differently from the protobuf deserializer.